### ISSUE TITLE
icon highlight on changing space broken on sketchybar-v1.4.0

### ISSUE THREAD

**Issue by ut0mt8** [FelixKratz/SketchyBar#76]
Maybe I've forgot something but seems since the upgrade that the icon of my space is not highlighted anymore ?
Any change in the config file that can explain this?

**Comment by ut0mt8** [comment]
well launching on foreground I have  : 

```
sh: -c: line 0: syntax error near unexpected token `fi'
sh: -c: line 0: `if [ "$SELECTED" = "true" ]; thensketchybar -m set $NAME icon_highlight on;elsesketchybar -m set $NAME icon_highlight off;fi'
sh: -c: line 0: syntax error near unexpected token `fi'
sh: -c: line 0: `if [ "$SELECTED" = "true" ]; thensketchybar -m set $NAME icon_highlight on;elsesketchybar -m set $NAME icon_highlight off;fi'
sh: -c: line 0: syntax error near unexpected token `fi'
sh: -c: line 0: `if [ "$SELECTED" = "true" ]; thensketchybar -m set $NAME icon_highlight on;elsesketchybar -m set $NAME icon_highlight off;fi'
sh: -c: line 0: syntax error near unexpected token `fi'
sh: -c: line 0: `if [ "$SELECTED" = "true" ]; thensketchybar -m set $NAME icon_highlight on;elsesketchybar -m set $NAME icon_highlight off;fi'

````

related in one way or another.

**Comment by FelixKratz** [comment]
Fixed in fb254593464988198100206bc2f7353988360b93, sorry blundered that in the restructure and didnt notice because I was using a custom space script.
